.github/workflows/integration_tests.yml

```diff
@@ -41,39 +41,6 @@ jobs:
         with:
           path: dev/rucio
           fetch-depth: 0
-      - name: Select tag for rucio containers
-        shell: bash
-        run: | 
-          # Change to cloned rucio/rucio repo
-          cd $GITHUB_WORKSPACE/dev/rucio
-          
-          # Get current branch and corresponding latest tag in rucio/rucio repo
-          BRANCH=$(git rev-parse --abbrev-ref HEAD)
-          OWNER="${{ github.repository_owner }}"
-          
-          if [ $OWNER != 'rucio' ]; then
-            echo "The workflow is running in user ${OWNER}'s fork. Fetching branches and tags from rucio/rucio instead."
-            git remote add rucio https://github.com/rucio/rucio
-            git fetch rucio --tags
-          fi
-
-          echo "On branch ${BRANCH}"
-          if [ $BRANCH == 'master' ]; then
-              GIT_REF="master"
-          else
-              GIT_REF=$(git describe --tags --abbrev=0)
-              echo "Latest tagged release on branch $BRANCH is $GIT_REF"
-          fi
-
-          cd $GITHUB_WORKSPACE
-
-          # Check if rucio/containers has a corresponding $GIT_REF 
-          if [ $(git tag -l "$GIT_REF") ]; then
-              git checkout tags/$GIT_REF
-          else
-              echo "Tag $GIT_REF not found in rucio/containers. Integration test containers will be built against the master branch instead."
-              git checkout master
-          fi
```

Comment:
Why is this deleted?
As far as I know, this was included (#5256) since changes made to the rucio/containers repo were only used when a new container was released. However, we want to use these changes in the tests immediately.

Reply:
It needs to be patched. I will send a separate PR later today.  ;)